spec/support/shared/shared_let_env_hash.rb

```diff
@@ -0,0 +1,3 @@
+RSpec.shared_context 'shared lets', :a => :b do
```

Comment:
Err now, this was voodoo troubleshooting from when i had a misplaced _ in one usage of the shared context name that i couldn't see due to Rubymine's spellcheck highlighting.